client/packages/requisitions/src/ResponseRequisition/DetailView/IndicatorEdit/IndicatorLineEdit.tsx

```diff
@@ -54,7 +54,7 @@ const InputWithLabel = ({ data }: { data: IndicatorColumnNode }) => {
     data.valueType === IndicatorValueTypeNode.Number ? (
       <NumericTextInput
         width={INPUT_WIDTH}
-        value={Number(draft?.value)}
+        value={isNaN(Number(draft?.value)) ? 0 : Number(draft?.value)}
         onChange={v => {
           update({ value: String(v) }).then(errorHandler);
```

Comment:
When holding down backspace to clear the value, it flickers no value and `0`. Then the back end is still getting requests of something causing this error: 
![image](https://github.com/user-attachments/assets/7d6690cc-6460-4af6-9c54-d340bf4be8ef)

![image](https://github.com/user-attachments/assets/cef35c40-09bf-493e-89d2-505c8a6b6343)

Undefined... weird. OK if you want to fix that one in a separate issue but may as well tackle in this PR if easy? Is the `v` in the handler function here undefined?